packages/core/strapi/src/cli/commands/cloud/utils/compress-files.ts

```diff
@@ -0,0 +1,90 @@
+import * as fs from 'fs';
+import * as tar from 'tar';
+import * as path from 'path';
+import { minimatch } from 'minimatch';
+
+const IGNORED_PATTERNS = [
```

Comment:
might be worth writing how this array is made i.e. where does it come from, because we'll need to maintain it and understand the reasoning behind it :)

Reply:
This array is a list of useless files for the cloud build. As users can create a strapi project without storing it in a Git repo, we cannot rely solely on .gitignore.
This is an arbitrary list I made to avoid sending 3go of useless data to our build-engine.